test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+describe RecipesController do
+  describe "search" do
+    it "succeeds" do
+      get root_path
+
+      must_respond_with :success
+    end
+  end
+
+  describe "index" do
+    it "succeeds for a valid search query" do
+      VCR.use_cassette('recipes') do
+        get results_path, params: { query: "chicken"}
```

Comment:
It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?